cluster/ci/config/prow/plugins.yaml

```diff
@@ -555,6 +555,10 @@ config_updater:
       name: prow-job-cluster-launch-e2e-azure-upgrade
       additional_namespaces:
       - ci-stg
+    ci-operator/templates/openshift/openshift-azure/rbac.yaml:
+      name: prow-job-rbac-azure
+      additional_namespaces:
+      - ci-stg
```

Comment:
Is this new for us?  Are we using a namespace as a staging environment?

Reply:
we can. I can show in the demo. its newish thing :)